GetRecordings/modules/extractor.py

```diff
@@ -102,20 +102,20 @@ def update_metadata(self):
         print(f'Metadata entries: {len(metadata)}')
         print(f'Database entries: {len(is_valid_new)}')
 
-        update_count:int = 0
-        for id in tqdm(metadata.index):
-            prev_value = metadata.at[id, 'is_valid']
-            new_value = str(is_valid_new.at[int(id), 'is_valid'])
+        with open('update_log.txt', 'a') as f_out:
```

Comment:
Moved open up here so that we don't have to open and close it thousands of times xD